tests/functional/disaster-recovery/regional-dr/test_protected_application_list_view.py

```
@@ -0,0 +1,536 @@
+        logger.info("=" * 60)
+        logger.info("Step 1: Verify applications on Protected Applications page")
+        logger.info("=" * 60)
+        try:
+            verify_protected_applications_list_view(
+                acm_obj=acm_obj,
+                appset_workloads=appset_workloads,
+                discovered_workloads=discovered_workload,
+                timeout=120,
+            )
+            logger.info("Step 1 PASSED: All applications verified on Protected Applications page")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)